src/mainview.cpp

```diff
@@ -37,23 +53,23 @@ CMainView::CMainView( QWidget *pParent ) :
 	}
 
 	QJsonObject JSONConfig = JSONConfigDocument.object();
-
 	for ( auto it = JSONConfig.begin(); it != JSONConfig.end(); it++ )
 	{
 		auto pHeader = new QLabel( it.key(), this );
 		pHeader->setObjectName( "Header" );
-		pLayout->addWidget( pHeader );
+		pLayout2->addWidget( pHeader );
 		QJsonArray arr = it.value().toArray();
-		for ( int i = 0; i < arr.size(); i++ )
+
+		for (auto && i : arr)
```

Comment:
why `&&`? did you mean just `&`?

Reply:
It was something my code correction suggested.

Reply:
I got the answer:
> Non-const lvalue reference to type 'QJsonValueRef' cannot bind to a temporary of type 'QJsonValueRef'

Reply:
Well you could also do `for ( const auto &i : arr )` for it instead

Reply:
Is there a real difference?

Reply:
Not too sure honestly, but I like the look of const rather than just &&

Reply:
All righty.